lib/ssh_scan/database.rb

```diff
@@ -0,0 +1,26 @@
+# API Databases
+require 'ssh_scan/database/mongo'
+require 'ssh_scan/database/sqlite'
+
+module SSHScan
+  class DatabaseConfig
+    def initialize(opts = {})
+      @@db_type = opts['database']['type'] || 'mongodb'
+    end
+
+    def self.from_config_file
+      @@db_path = './lib/ssh_scan/database/database_config.yml'
```

Comment:
It should also use the expand path pattern to load it from a predictable location when it's bundled as a Gem.  Search for expand in the source tree and you'll find an example.

I'm also thinking that maybe we need a common location for configuration files.  Perhaps ./config and land them all there.